PENDING_RELEASE_NOTES.md

```diff
@@ -3,10 +3,16 @@
 _Date TBD_
 
 ### View/Table modifications
+>>>>>>> development
 * Create table 'filter' to handle storing user's filters
-* Create table 'filter_type' to indicate the type of filter 
+* Create table 'filter_type' to indicate the type of filter
+
+### View Modifications
```

Comment:
Not sure why we have "View/Table modifications" separate from "View" modifications. Should either be combined or split